design-library/src/components/BccSpinner/BccSpinner.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+type Props = {};
+
+withDefaults(defineProps<Props>(), {});
+</script>
+<template>
+  <svg
+    class="... mr-3 h-5 w-5 animate-spin"
```

Comment:
As mentioned before, these classes should be moved to the `bcc-spinner` class with `@apply` in the `BccSpinner.css` file.

Reply:
Also it seems many of these classes are not needed. I think the developers using the spinner would want to choose a size for it themselves so the `h-5 w-5` classes should be removed.

Also the `... mr-3` classes are not needed in general. Only the `animate-spin` class is actually necessary here